### PR TITLE
Body footer

### LINKED ISSUE #74: Footer for all pages (1)
As a Nova, I need to have access to various sections of OPL from the bottom of the page so that I can always access practices and other details of the site. 

Footer
**Given** I am on any page of the app
**When** I can access footer pages and external links
**Then** I see ways to navigate the site
**And** I can click on external links.

NOTES:
- See global footer in Home Page design here - https://zpl.io/aBJ4L0m
- Assets are in Zeplin for download please use the colored Labs logo
- Email link (TBD) please ping @heyjerrybecker to determine
- External links include: 
Podcast https://podcasts.apple.com/us/podcast/open-practice-podcast/id1501715186
Insta https://www.instagram.com/openpracticelibrary/?hl=en
Git https://github.com/openpracticelibrary
YT https://www.youtube.com/channel/UCBGbB7xVsmf-yQ8NXn7TOVg?view_as=subscriber

Screen Shot 2020-05-29 at 11 53 19 AM

BLOCKERS:
No

### PR DESCRIPTION
**Pull request by piyushsn**
Resolves #74 
- Updated logo and links as per design in Zeplin.

### CODE REVIEW

File: src/components/shared/BodyFooter/FooterLinkSection.js [hunk 1]
```diff
@@ -39,21 +40,45 @@ export default function FooterLogoSection() {
...
-            </Typography>
-          ))}
+          <Grid
+            container 
+            direction="row" 
+            spacing={5} 
+          >
+            {sections.map((section, i) => (
+              <Grid item>
+                <Link href={section.url}>
```
Comment (shaheinm) [inline src/components/shared/BodyFooter/FooterLinkSection.js:71]: Don't use `<Link>`, use the `navigate()` function as it was before.

Comment (piyushsn) [reply src/components/shared/BodyFooter/FooterLinkSection.js]: changes done :)

**Comment by shaheinm** [comment]
You need to update the snapshot for the footer test

**Comment by shaheinm** [comment]
I added it for you here, but don't forget to do the `Resolves` keyword link back to the issue :)

File: src/components/practicePage/PageIntro/__tests__/__snapshots__/PageIntro.test.js.snap [hunk 2]
```diff
@@ -174,10 +174,10 @@ exports[`matches the snapshot 1`] = `
               data-testid="dates"
             >
               Published 
-              June 5, 2020
+              June 8, 2020
                | Last edited
                
-              June 5, 2020
+              June 8, 2020
             </span>
```
Comment (lallmon) [inline src/components/practicePage/PageIntro/__tests__/__snapshots__/PageIntro.test.js.snap:181]: Is this a dynamic date?

Comment (piyushsn) [reply src/components/practicePage/PageIntro/__tests__/__snapshots__/PageIntro.test.js.snap:181]: yeah, it get updates during test, I think so ....

Comment (lallmon) [reply src/components/practicePage/PageIntro/__tests__/__snapshots__/PageIntro.test.js.snap:181]: Not a change request or anything, but that will mean this snapshot will fail every time the file is touched, not just when the rendered output changes.

**Pull request merged**